crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -22,32 +22,32 @@ class Api::UpgradeController < ApiController
   api_version "2.0"
   example '
   {
-    "current_step": "admin_upgrade",
+    "current_step": "admin",
     "current_substep": null,
     "current_node": null,
     "remaining_nodes": null,
     "upgraded_nodes": null,
     "steps": {
-      "upgrade_prechecks": {
+      "prechecks": {
         "status": "passed",
         "errors": {}
       },
-      "upgrade_prepare": {
+      "prepare": {
         "status": "passed",
         "errors": {}
       },
-      "admin_backup": {
+      "backup_crowbar": {
         "status": "passed",
         "errors": {}
       },
-      "admin_repo_checks": {
+      "repocheck_crowbar": {
         "status": "passed",
         "errors": {}
       },
-      "admin_upgrade": {
+      "admin": {
```

Comment:
ah now i get it... makes sense somehow.
but on the other side having crowbar and openstack in the backup subcommand also makes sense, plus there is also node data in there, not just admin related stuff...